happybase/table.py

```diff
@@ -222,7 +222,7 @@ def cells(self, row, column, versions=None, timestamp=None,
     def scan(self, row_start=None, row_stop=None, row_prefix=None,
              columns=None, filter=None, timestamp=None,
              include_timestamp=False, batch_size=1000, scan_batching=None,
-             limit=None, sorted_columns=False):
+             limit=None, sorted_columns=False, reversed=False):
```

Comment:
Fair point, I had forgotten about the built-in `reversed`. As it stands today, filter argument also shadows built-in `filter` though.